pkg/sdn/plugin/master.go

```diff
@@ -27,6 +27,7 @@ type OsdnMaster struct {
 
 func StartMaster(networkConfig osconfigapi.MasterNetworkConfig, osClient *osclient.Client, kClient *kclient.Client) error {
 	if !osapi.IsOpenShiftNetworkPlugin(networkConfig.NetworkPluginName) {
+		cleanupOpenShiftNetworkArtifacts(osClient)
```

Comment:
@smarterclayton Agreed. Thanks Clayton! 
Do you mean that we should record an event in some way like `recorder.Eventf`? If so, where will the event be seen by users? I'm not familiar with the event record mechanism, sorry:) Is just logging an error enough? It seems that there are no other events recorded in master starting process.